helix-core/src/test/java/org/apache/helix/TestListenerCallback.java

```diff
@@ -119,6 +119,8 @@ public void beforeClass() throws Exception {
 
   @AfterClass
   public void afterClass() throws Exception {
+    String testClassName = this.getShortClassName();
```

Comment:
Let us make this name consistent with line 102-103, i.e, either fix that or change this line to make them consistent.

Reply:
changed.

Reply:
actually, remove this one as it is moved to zkTestBase.

Reply:
remove this one and as logging from ZkTestBase now.